src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -360,10 +360,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Function
 
         ''' <summary>
+        ''' Does symbol or its containing type have Microsoft.CodeAnalysis.Embedded() attribute
+        ''' </summary>
+        <Extension()>
+        Friend Function IsHiddenByCodeAnalysisEmbeddedAttribute(symbol As Symbol) As Boolean
```

Comment:
>IsHiddenByCodeAnalysisEmbeddedAttribute [](start = 24, length = 39)

It looks like significant portion of this function can be shared with IsHiddenByVisualBasicEmbeddedAttribute.